controls/def.cf

```diff
@@ -196,6 +196,14 @@ bundle common def
       comment => "Define plugins path",
       handle => "common_def_vars_dir_plugins";
 
+      "dir_templates"
```

Comment:
I don't understand why only this variable needs the `mpf_` prefix. Is it used anywhere else?